region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -573,6 +535,36 @@
       ],
       "note": "Use the Fireflea when it moves to the right to cross the first gap."
     },
+    {
+      "link": [2, 3],
+      "name": "HiJump and Wall Jump",
+      "requires": [
+        "HiJump",
+        "canPreciseWalljump",
+        "canIframeSpikeJump",
+        {"spikeHits": 2}
```

Comment:
Jumping on the fire flea after the walljump can save some health.  Not really well known/ intuitive though.